db/migrate/20230725143607_add_confirmable_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddConfirmableToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :confirmation_token, :string
+  end
+end
```

Comment:
- **[OPTIONAL]** This file causes migration errors when migrating the db. This file is not needed. Kindly remove it.